src/DataObjects/AbstractCmisObject.php

```diff
@@ -747,7 +747,11 @@ public function setAcl(array $aces)
      */
     public function getAcl()
     {
-        return $this->acl;
+        $result = $this->getSession()->getAcl($this, false);
+
+        $this->refresh();
+
+        return $result;
```

Comment:
i test it again and it will not work when i remove this code. acl stuff comes with explicit call "getAcl" and is not part of object cmisselector - maybe this is the reason why.

Reply:
@sascha-egerer are you able to test this code in your environment?

Reply:
@aWuttig Sorry i'm not using this lib and so i don't have an environment (the lib was developed for the ForgetIT Project and is not actively maintained). I have to setup one to test that change. Will give you feedback as soon as i could test it.

Reply:
I really don't get why this is required in your case. Could you please test it again if you really need the refresh?
I wrote an example file and it works fine without this refresh.